Userland/Libraries/LibC/sys/arch/aarch64/regs.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022, Konrad <konradek@github.com>
```

Comment:
Do you have an actual github.com email address? This looks unusual.

Reply:
hahah! yup! is it okay to use <konrad@serenityos.org> without having mail (alias) there? 😝